libreyolo/models/sam/prompts.py

```
@@ -0,0 +1,176 @@
+        if labels is not None:
+            raise ValueError("labels given without points.")
+        return None
+    if labels is None:
+        return [[1] * len(obj) for obj in canonical_points]
+
+    lbls = _to_list(labels)
+    d = _depth(lbls)
+    if d not in (1, 2):
+        raise ValueError(
+            f"labels must be [l, ...] or [[l, ...], ...]; got nesting depth {d}."
+        )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)